pixiedust/services/stashCloudant.py

```diff
@@ -34,8 +34,8 @@ def doRender(self, handlerId):
 
         config = ShellAccess.sc._conf.getAll()
         if not any("spark.jars" in s for s in config):
-            self._addHTML("Please set PYSPARK_SUBMIT_ARGS to --jars local_dir_path/cloudant-spark.jar in kernel.json")
-            self.debug("Please set PYSPARK_SUBMIT_ARGS to --jars <local_dir_path>/cloudant-spark.jar in kernel.json")
+            self._addHTML("Please setup one time manual steps to configure stashing to Cloudant database at https://ibm-cds-labs.github.io/pixiedust/install.html#stash-to-cloudant")
```

Comment:
You should pass well formed html to _addHTML e.g
<div>Please setup one time manual steps to configure stashing to Cloudant data at <a href="https://ibm-cds-labs.github.io/pixiedust/install.html#stash-to-cloudant">https://ibm-cds-labs.github.io/pixiedust/install.html#stash-to-cloudant</a></div>